.circleci/config.yml

```diff
@@ -217,12 +227,10 @@ workflows:
             - nancy/test
             - test
             - golangci/lint
-            - test-e2e-memory
             - test-e2e-postgres
             - test-e2e-mysql
             - test-e2e-cockroach
             - test-e2e-plugin
-            - test-legacy-migrations-postgres
```

Comment:
Why did you remove these?

Reply:
Because it still depend on indirectly. `test-legacy-migrations-mysql` and `test-legacy-migrations-cockroach` depend on `test-legacy-migrations-postgres`. So does `test-e2e-memory`.